### PR TITLE
Bump version to 6.10.1

### PR DESCRIPTION
**Pull request by axue-broadcom**

### CODE REVIEW

File: .github/workflows/create-release-on-tag.yml [hunk 1]
```diff
@@ -22,14 +22,15 @@ jobs:
...
 
-            ### Enhancements:
+            ### Bug Fixes:
 
-            - [#1080](https://github.com/greenplum-db/pxf/pull/1080) Add support for UUID to JDBC profile
-            - [#1081](https://github.com/greenplum-db/pxf/pull/1081) Add automation test cases for JDBC profile DATE_WIDE_RANGE feature
-            - [#1093](https://github.com/greenplum-db/pxf/pull/1093) Bump Postgresql JDBC driver to 42.7.2
-            - [#1096](https://github.com/greenplum-db/pxf/pull/1096) Add formatting for Date values
+            - [#1100](https://github.com/greenplum-db/pxf/pull/1100) Refactor position tracking for multi-line JSON files
+            - [#1102](https://github.com/greenplum-db/pxf/pull/1102) bumped gp-common-go-libs to v1.0.16
```
Comment (tatb-vmware) [inline .github/workflows/create-release-on-tag.yml:30]: Nit
```suggestion
            - [#1102](https://github.com/greenplum-db/pxf/pull/1102) Bumped gp-common-go-libs to v1.0.16
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 6.10.1 (03/27/2024)
+
+### Bug Fixes:
+
+- [#1100](https://github.com/greenplum-db/pxf/pull/1100) Refactor position tracking for multi-line JSON files
+- [#1102](https://github.com/greenplum-db/pxf/pull/1102) bumped gp-common-go-libs to v1.0.16
```
Comment (tatb-vmware) [inline CHANGELOG.md:8]: Nit
```suggestion
- [#1102](https://github.com/greenplum-db/pxf/pull/1102) Bumped gp-common-go-libs to v1.0.16
```

**Review by tatb-vmware — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (axue-broadcom) [reply CHANGELOG.md:8]: The original PR does not have this upper cased. I can do it here for consistency but we try to keep things matching with the PR title for the changelog.

Comment (axue-broadcom) [reply CHANGELOG.md:8]: Ah I see. The commit title was lower cased but the PR title was upper cased. I will uppercase it here.

**Review by yimingli-vmware — APPROVED** [review]

**Pull request merged**